From 7e95ea6f071b17a40c266f54b9b01f3da6dc05cf Mon Sep 17 00:00:00 2001
From: liamhuber <liamhuber@greyhavensolutions.com>
Date: Mon, 22 Jul 2024 09:37:14 -0700
Subject: [PATCH] [patch] Import annotations for 3.9 compatibility with type
 hint sugar

---
 pyiron_snippets/resources.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/pyiron_snippets/resources.py b/pyiron_snippets/resources.py
index cc17918..5a6e3e8 100644
--- a/pyiron_snippets/resources.py
+++ b/pyiron_snippets/resources.py
@@ -2,6 +2,8 @@
 Classes to find data files and executables in global paths.
 """
 
+from __future__ import annotations
+
 from abc import ABC, abstractmethod
 from collections.abc import Iterator, Iterable
 import os
